lib/board.rb

```diff
@@ -0,0 +1,50 @@
+class Board
+  attr_accessor :cells
+
+  @cells = []
```

Comment:
you're right, I realize that I create the cells array in initialize. So line 4 is unneccessary.